components/feature/downloads/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
```

Comment:
Make sure to add this file to l10n.toml so that those strings get translated:
https://github.com/mozilla-mobile/android-components/blob/master/l10n.toml

Reply:
👍

Reply:
Done 👍 
https://github.com/mozilla-mobile/android-components/blob/cb34bb9b8d9ceafd5a9d03fe65a7099c5b80efb1/l10n.toml#L25